docs/_interface/Private-Network.md

```diff
@@ -0,0 +1,291 @@
+  "difficulty": "1",
+  "gasLimit": "8000000",
+  "extradata": "0x00000000000000000000000000000000000000000000000000000000000000007df9a875a174b3bc565e6424a0050ebc1b2d1d820000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000",
+  "alloc": {
+    "7df9a875a174b3bc565e6424a0050ebc1b2d1d82": { "balance": "300000" },
+    "f41c74c9ae680c1aa78f42e5647a62f353b7bdde": { "balance": "400000" }
+  }
+}
+```
+
+#### Ethash Example
+
+Since ethash is the default consensus algorithm, no additional parameters need to be
+configured in order to use it. You can influence the initial mining difficulty using the
+`difficulty` parameter, but note that the difficulty adjustment algorithm will quickly
+adapt to the amount of mining resources you deploy on the chain.
+
+```json
+{
+  "config": {
+    "chainId": 15,
+    "homesteadBlock": 0,
+    "eip150Block": 0,
+    "eip155Block": 0,
+    "eip158Block": 0,
+    "byzantiumBlock": 0,
+    "constantinopleBlock": 0,
+    "petersburgBlock": 0,
+    "ethash": {}
+  },
+  "difficulty": "1",
+  "gasLimit": "8000000",
+  "alloc": {
+    "7df9a875a174b3bc565e6424a0050ebc1b2d1d82": { "balance": "300000" },
+    "f41c74c9ae680c1aa78f42e5647a62f353b7bdde": { "balance": "400000" }
+  }
+}
+```
+
+### Initializing the Geth Database
+
+To create a blockchain node that uses this genesis block, run the following command. This
+imports and set the canonical genesis block for your chain.
+
+```shell
+geth init --datadir data genesis.json
+```
+
+Future runs of geth using this data directory will use the genesis block you have defined.
+
+```shell
+geth --datadir data --networkid 15
+```
+
+### Scheduling Hard Forks
+
+As Ethereum protocol development progresses, new Ethereum features become available. To
+enable these features on your private network, you must schedule a hard fork.
+
+First, choose any future block number where the hard fork will activate. Continuing from
+the genesis.json example above, let's assume your network is running and its current block
+number is 35421. To schedule the 'Istanbul' fork, we pick block 40000 as the activation
+block number and modify our genesis.json file to set it:
+
+```json
+{
+  "config": {
+    ...
+    "istanbulBlock": 40000,
+    ...
+  },
+  ...
+}
+```
+
+In order to update to the new fork, first ensure that all Geth instances on your private
+network actually support the Istanbul fork (i.e. ensure you have the latest version of
+Geth installed). Now shut down all nodes and re-run the `init` command to enable the new
+chain configuration:
+
+```shell
+geth init --datadir data genesis.json
+```
+
+### Setting Up Networking
+
+Once your node is initialized to the desired genesis state, it is time to set up the
+peer-to-peer network. Any node can be used as an entry point. We recommend dedicating a
+single node as the rendezvous point which all other nodes use to join. This node is called
+the 'bootstrap node'.
+
```

Comment:
```suggestion

#### Setting up a bootstrap node
```

Reply:
I'm actually trying to remove these micro-headings as much as possible because they hurt readability.

Reply:
It absolutely makes sense. I just wanted to be make it more explicit that setting up the network is  two-step process, first set up the bootnode, then the member nodes.